gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/QueryAndGatewayDtoConverter.java

```diff
@@ -461,6 +461,8 @@ public static Filter addTimeAndSpaceFiltersAndConvertToQueryFilter(
       String spacesAttributeId,
       org.hypertrace.gateway.service.v1.common.Filter providedFilter) {
 
+    timestampAttributeId = timestampAttributeId.concat("Filter");
```

Comment:
What is the config for this?

Reply:
config I have updated here: https://github.com/razorpay/hypertrace-query-service/pull/30/files